CMake/Modules/python/PythonLibs.cmake

```diff
@@ -11,7 +11,6 @@
 find_package(PythonLibs 2.7 REQUIRED)
 
 include_directories(${PYTHON_INCLUDE_DIRS})
-flow_define(HAVE_PYTHON)
```

Comment:
`flow_define(HAVE_PYTHON)` is still present in `CMakeLists.txt` no change required